packages/web-react/src/components/Alert/useAlertIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import { useIconName } from '../../hooks/useIconName';
+import { SpiritAlertProps } from '../../types';
+
+export function useAlertIcon(props: SpiritAlertProps) {
```

Comment:
Maybe this should also work.

```suggestion
export function useAlertIcon({ color, iconName }: SpiritAlertProps) {
```